lib/utils/webpack.config.js

```diff
@@ -117,6 +117,7 @@ module.exports = (program, directory, stage, webpackPort = 1500, routes = []) =>
             },
             __PREFIX_LINKS__: program.prefixLinks,
           }),
+          new ExtractTextPlugin('styles.css'),
```

Comment:
I did some testing: for `build-html` step css-modules loaders has to be turned on (or just not turned off), they use `ExtractTextPlugin` so it also has to be included in the build step configuration. On the other hand they are not needed in `build-javascript` step. I don't know why previously it seemed to me that this is necessary.